test/builds/build_and_test.yml

```diff
@@ -133,3 +134,209 @@ jobs:
 
   - script: rm -rf *
     displayName: 'Clean'
+
+
+- job: 'RLO_Ubuntu'
```

Comment:
Is it possible to "#include" the corresponding file from rlo/test/builds ? It'll change with e.g. ksc Expr